kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ActiveAddFilesIterator.java

```diff
@@ -103,33 +112,37 @@ public void close() throws IOException {
 
     /**
      * Grabs the next FileDataReadResult from `iter` and updates the value of `next`.
-     *
+     * <p>
```

Comment:
do we need `<p>` etc in `/internal/` code? we won't be publishing javadocs for this

Reply:
Intellij auto-format adds these. It replaces the newline in javadocs with <p>.